docs/contributing-analyzer.md

```diff
@@ -17,28 +17,33 @@ In general, it is best to run commands from the Visual Studio Developer Command
     - ASP.NET and web development
     - .NET desktop development
     - Visual Studio extension development
-1. Ensure to install Individual components:
+1. Ensure to install *Individual components*:
     - .NET Framework 4.8 SDK
     - .NET Framework 4.8 Targeting pack
+    - .NET Framework 4.7.2 Targeting pack
+    - .NET Framework 4.6 Targeting pack
     - .NET SDK
     - .NET Compiler Platform SDK
 1. Install also:
-    - .NET Core 3.1 SDK
-    - .NET 3.5 SDK (SP1)
+    - [.NET Core 3.1 SDK](https://dotnet.microsoft.com/en-us/download/dotnet)
+    - .NET 3.5 SDK (SP1) TODO: Add link to dotNetFx35setup.exe
```

Comment:
can you add it, so we can merge this PR? :)

Reply:
@pavel-mikula-sonarsource do you know where to get this from?

Reply:
https://www.microsoft.com/en-us/download/details.aspx?id=21